nnpdf_data/nnpdf_data/commondata/CMS_Z0J_13TEV_PT/kinematics.yaml

```diff
@@ -0,0 +1,1141 @@
+    max: null
+- pT:
+    min: 52.0
+    mid: 58.5
+    max: 65.0
+  m_ll2:
+    min: null
+    mid: 8317.44
+    max: null
+  sqrts:
+    min: null
+    mid: 13000.0
+    max: null
+- pT:
+    min: 65.0
+    mid: 75.0
+    max: 85.0
+  m_ll2:
+    min: null
+    mid: 8317.44
+    max: null
+  sqrts:
+    min: null
+    mid: 13000.0
+    max: null
+- pT:
+    min: 85.0
+    mid: 102.5
+    max: 120.0
+  m_ll2:
+    min: null
+    mid: 8317.44
+    max: null
+  sqrts:
+    min: null
+    mid: 13000.0
+    max: null
+- pT:
+    min: 120.0
+    mid: 140.0
+    max: 160.0
+  m_ll2:
+    min: null
+    mid: 8317.44
+    max: null
+  sqrts:
+    min: null
+    mid: 13000.0
+    max: null
+- pT:
+    min: 160.0
+    mid: 175.0
+    max: 190.0
+  m_ll2:
+    min: null
+    mid: 8317.44
+    max: null
+  sqrts:
+    min: null
+    mid: 13000.0
+    max: null
+- pT:
+    min: 190.0
+    mid: 205.0
+    max: 220.0
+  m_ll2:
+    min: null
+    mid: 8317.44
+    max: null
+  sqrts:
+    min: null
+    mid: 13000.0
+    max: null
+- pT:
+    min: 220.0
+    mid: 235.0
+    max: 250.0
+  m_ll2:
+    min: null
+    mid: 8317.44
+    max: null
+  sqrts:
+    min: null
+    mid: 13000.0
+    max: null
+- pT:
+    min: 250.0
+    mid: 275.0
+    max: 300.0
+  m_ll2:
+    min: null
+    mid: 8317.44
+    max: null
+  sqrts:
+    min: null
+    mid: 13000.0
+    max: null
+- pT:
+    min: 300.0
+    mid: 325.0
+    max: 350.0
+  m_ll2:
+    min: null
+    mid: 8317.44
+    max: null
+  sqrts:
+    min: null
+    mid: 13000.0
+    max: null
+- pT:
+    min: 350.0
+    mid: 375.0
+    max: 400.0
+  m_ll2:
+    min: null
+    mid: 8317.44
+    max: null
+  sqrts:
+    min: null
+    mid: 13000.0
+    max: null
+- pT:
+    min: 400.0
+    mid: 425.0
+    max: 450.0
+  m_ll2:
+    min: null
+    mid: 8317.44
+    max: null
+  sqrts:
+    min: null
+    mid: 13000.0
+    max: null
+- pT:
+    min: 450.0
+    mid: 475.0
+    max: 500.0
+  m_ll2:
+    min: null
+    mid: 8317.44
+    max: null
+  sqrts:
+    min: null
+    mid: 13000.0
+    max: null
+- pT:
+    min: 500.0
+    mid: 750.0
+    max: 1000.0
+  m_ll2:
+    min: null
+    mid: 8317.44
+    max: null
+  sqrts:
+    min: null
+    mid: 13000.0
+    max: null
+- pT:
+    min: 0.0
+    mid: 1.0
+    max: 2.0
+  m_ll2:
+    min: null
+    mid: 8317.44
+    max: null
+  sqrts:
+    min: null
+    mid: 13000.0
+    max: null
+- pT:
+    min: 2.0
+    mid: 3.0
+    max: 4.0
+  m_ll2:
+    min: null
+    mid: 8317.44
+    max: null
+  sqrts:
+    min: null
+    mid: 13000.0
+    max: null
+- pT:
+    min: 4.0
+    mid: 5.0
+    max: 6.0
+  m_ll2:
+    min: null
+    mid: 8317.44
+    max: null
+  sqrts:
+    min: null
+    mid: 13000.0
+    max: null
+- pT:
+    min: 6.0
+    mid: 7.0
+    max: 8.0
+  m_ll2:
+    min: null
+    mid: 8317.44
+    max: null
+  sqrts:
+    min: null
+    mid: 13000.0
+    max: null
+- pT:
+    min: 8.0
+    mid: 9.0
+    max: 10.0
+  m_ll2:
+    min: null
+    mid: 8317.44
```

Comment:
I'm afraid something went wrong here since all data is set as mll2 == 8317.44, mll ~ 91 but this dataset should be separated by ranges of mll